CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+## 0.9.0
+
+* Changelog started, changes from previous release `0.8.0`:
+
+* Major breaking:
+  * Removed instances due to migration to `haskeline >= 0.8 && < 0.9`:
+    * `instance MonadException m => MonadException(StateT(HashMap FilePath NExprLoc) m)`
+    * `instance MonadException m => MonadException(Fix1T StandardTF m)`
+
+* Minor:
```

Comment:
Do we?

Reply:
The `haskeline` update allowed us to build with GHC 8.10 too. I'd call the [`liftTyped` issue](https://github.com/haskell-nix/hnix/issues/592#issuecomment-634665426) a known bug – it won't affect existing code though. Do you think we should point that out in the changelog?

Reply:
CI never successfully compiled a project with GHC 8.10, so we can not claim project support of it, until every PR/release is not checked to comply with the GHC 8.10.

Reply:
> CI never successfully compiled a project with GHC 8.10

I thought that was just a nixpkgs problem, not a property of the Cabal package that we're publishing here.

Reply:
Honesly, since other people who already know the theme are on that issue, I did not looked into it.

In reality, Nixpkgs current default compiler is `GHC 8.8.3`, so all `haskellPackages` infrastructure is suited for it, no wonder it does not support `GHC 8.10`. It was my main version, but I do not looked into it, and I still didn't looked and understood/found the cause, so reserved judgment.

The proper way to solve this - is to have a separate build that does how project states in readme. `nix-shell -> cabal v2-build` - that would be a quite other build from what CI did&does (`nix-build`).
And build without Nix only with Cabal - is the other build.

And then the question why we have all those Nix files and being an Nix interpreter, but those arguments does not hold to the core issue that this is in truth the Haskell project first, and it is Nix interpreter is the second here.

Reply:
We should have some basic Cabal build, that way we would see where we have the Nixpkgs issue, you feel here should be multicheck that reports that something in fact works, but hits a Nixpkgs code somewhere?

Reply:
Again, only when GHC 8.10 checks in CO, I can claim its support, because only then I would be able to merge the code only when that code supports GHC 8.10.